ui/imports/shared/popups/walletconnect/private/ContextCard.qml

```diff
@@ -0,0 +1,87 @@
+import QtQuick 2.15
```

Comment:
Note that @Seitseman already extracted these in his PR.

Reply:
ohh, right! In this case I'll rebase on top of his PR